codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectEncoder.java

```diff
@@ -195,7 +195,14 @@ private void encodeChunkedContent(ChannelHandlerContext ctx, Object msg, long co
         }
     }
 
-    boolean isContentAlwaysEmpty(@SuppressWarnings("unused") H msg) {
+    /**
+     * Determine whether a message has a content or not. Some message may have headers indicating
+     * a content without having an actual content, e.g the response to an HEAD or CONNECT request.
+     *
+     * @param msg the message to test
+     * @return true to signal the message has no content
```

Comment:
I tend to agree with @jodastephen that `{@code ...}` is overhead for `true`, `false`, and `null`. 😄